util/runtime.go

```diff
@@ -3,11 +3,16 @@ package util
 import (
 	"runtime"
 
-	_ "go.uber.org/automaxprocs"
+	"go.uber.org/automaxprocs/maxprocs"
 )
 
-// Automaxprocs automatically set GOMAXPROCS to match Linux container CPU quota.
-// So we are wrapping it here to make sure we do not call it anywhere else without importing automaxprocs.
+func init() {
+	// Disable maxprocs logs
+	_, _ = maxprocs.Set(maxprocs.Logger(func(_ string, _ ...any) {}))
```

Comment:
You don't need to pass any maxprocs.Logger() function.

_, _ = maxprocs.Set()

Should work just as well.